### PR TITLE
Dont throw exceptions for illegal chars and keepalive queue mismatch

### PR DESCRIPTION
**Pull request by Outfluencer**
If a player get kicked for illegal chars or keepalive queue mismatch the full stacktrace will be printed

### CODE REVIEW

**Comment by Janmm14** [comment]
It would be probably better to change the exception throwing into a QuietException

**Comment by md-5** [comment]
Or just disconnect without throwing an exception

File: proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java [hunk 1]
```diff
@@ -147,7 +150,11 @@ public void handle(Chat chat) throws Exception
         for ( int index = 0, length = chat.getMessage().length(); index < length; index++ )
         {
             char c = chat.getMessage().charAt( index );
-            Preconditions.checkArgument( c != '\u00A7' && c >= ' ' && c != 127, "illegal characters in chat" ); // Section symbol, control sequences, and delete
+            if ( c == '\u00A7' || c < ' ' || c == 127 ) // Section symbol, control sequences, and delete
+            {
+                con.disconnect( "illegal characters in chat" );
```
Comment (MrIvanPlays) [inline proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java:155]: maybe a translation would be better here?

File: proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java [hunk 2]
```diff
@@ -131,7 +130,11 @@ public void handle(KeepAlive alive) throws Exception
 
         if ( keepAliveData != null && alive.getRandomId() == keepAliveData.getId() )
         {
-            Preconditions.checkState( keepAliveData == con.getServer().getKeepAlives().poll(), "keepalive queue mismatch" );
+            if ( keepAliveData != con.getServer().getKeepAlives().poll() )
+            {
+                con.disconnect( "keepalive queue mismatch" );
```
Comment (MrIvanPlays) [inline proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java:135]: perhaps explain to users what to do?

Comment (Janmm14) [reply proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java:135]: What the user has to do? Behave like a vanilla client.

Comment (MrIvanPlays) [reply proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java:135]: idk, this doesn’t give much information to the user

Comment (Janmm14) [reply proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java:135]: Now that I look at it again this does more look like a failsafe which should never happen

File: proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java (hunk 1 above)
Comment (Outfluencer) [reply proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java:155]: i don't think its needed. A vanilla client is not be able to send these chars

Comment (Outfluencer) [reply proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java:155]: and its the vanilla minecraft message for this case

File: proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java (hunk 2 above)
Comment (Outfluencer) [reply proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java:135]: Yes this should never happen so we don't need to give information about it

**Comment by Outfluencer** [comment]
I created new pull request to clean this up #3133

**Pull request closed without merging**